FogCreek/Glitch-Community
# Put links to pricing behind feature flag

## Links
* Remix link: https://sarah-staging.glitch.me/about
* Issue-tracker link: https://app.clubhouse.io/glitch/story/7085/put-footer-link-behind-feature-flag

## GIF/Screenshots:
![Screen Shot 2020-01-15 at 12 56 35 PM](https://user-images.githubusercontent.com/6620164/72458391-80939880-3796-11ea-9d2c-09f0e34da968.png)
![Screen Shot 2020-01-15 at 12 56 41 PM](https://user-images.githubusercontent.com/6620164/72458392-80939880-3796-11ea-9a17-1f6464e0184f.png)

## Changes:
Whoops, these links are not ready for prime time yet. 

## How To Test:
* If you don't have pufferfish enabled you should not be able to see links to pricing page on about in the nav or in the footer.
* To enable pufferfish you need to join https://glitch.com/@testing-team and then go to /secret and manually enable it

## Feedback I'm looking for:
any other links we could have forgotten?
